.devcontainer/devcontainer.json

```diff
@@ -30,5 +30,8 @@
     "remoteEnv": {
```

Comment:
I believe you should set `DOTNET_ROOT` and `PATH` also here so that they are inherited by the VS Code server.